pkg/broker/handler/processors/deliver/processor_test.go

```diff
@@ -321,6 +322,109 @@ func TestDeliverFailure(t *testing.T) {
 	}
 }
 
+type NoReplyHandler struct{}
+
+func (NoReplyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	w.WriteHeader(http.StatusAccepted)
+	// Read body to allow the connection to be reused.
+	io.Copy(ioutil.Discard, req.Body)
+	req.Body.Close()
+}
+
+func BenchmarkDeliveryNoReply(b *testing.B) {
+	sampleEvent := event.New()
+	sampleEvent.SetID("id")
+	sampleEvent.SetSource("source")
+	sampleEvent.SetSubject("subject")
+	sampleEvent.SetType("type")
+	sampleEvent.SetTime(time.Now())
+
+	deliverClient, err := ceclient.NewDefault()
+	if err != nil {
+		b.Fatalf("failed to create requester cloudevents client: %v", err)
+	}
+	targetSvr := httptest.NewServer(NoReplyHandler(struct{}{}))
+	defer targetSvr.Close()
+	ingressSvr := httptest.NewServer(NoReplyHandler(struct{}{}))
+	defer ingressSvr.Close()
+
+	broker := &config.Broker{Namespace: "ns", Name: "broker"}
+	target := &config.Target{Namespace: "ns", Name: "target", Broker: "broker", Address: targetSvr.URL}
+	testTargets := memory.NewEmptyTargets()
```

Comment:
nit: can directly user memory.NewTargets(target)

Reply:
This would require constructing the TargetsConfig first.